core/src/css/core.scss

```diff
@@ -127,3 +127,12 @@ html.plt-ios.plt-hybrid, html.plt-ios.plt-pwa {
     --ion-safe-area-right: env(safe-area-inset-right);
   }
 }
+
+
+.presenting-view-card {
```

Comment:
I hate that these are here. They are meant to be applied to a view/page that is presenting a modal, so we can apply a cool effect to scale it down. They are also iOS only.

Is there a per-platform CSS file for pages/views to put these in?